generic/tests/cfg/netperf.cfg

```diff
@@ -90,6 +91,12 @@
     env_setup_cmd += " echo 0 > /proc/sys/kernel/watchdog;"
     env_setup_cmd += " echo 0 > /proc/sys/kernel/nmi_watchdog;"
     env_setup_cmd += " setenforce 1"
+    bridge:
+        #physical_nic = p5p2
+    macvtap:
+        # please set netdst_nic1 to physica nic like netdst_nic1 = p5p2
+        nettype_nic1 = macvtap
```

Comment:
This does not make sense, and since you actually do no specify anything here, so from L#94 to L#99 could be removed.
Please check more about the configuration usage here: https://avocado-vt.readthedocs.io/en/latest/CartesianConfig.html
Thanks.